src/home/home.service.ts

```diff
@@ -79,4 +81,27 @@ export class HomeService {
       interests: interests,
     };
   }
+
+  async globalSearch(pagination: PaginationDto, query: HomeQuery) {
+    const [event, group] = await Promise.all([
+      this.eventService.searchAllEvents(pagination, query),
+      this.groupService.searchAllGroup(pagination, query),
```

Comment:
Looks good, but all nested fields are still pulled from db.
Make a separate methods `showGlobalSearchEvents` and `showGlobalSearchGroups` and `select` only 'name and slug' instead.